src/app/src/components/DashboardActivityReports.jsx

```diff
@@ -0,0 +1,365 @@
+    };
+
+    const updateReport = () => {
+        if (mode === REJECT) {
+            rejectReport({ id: selectedReport.id, statusChangeReason });
+        } else {
+            confirmReport({ id: selectedReport.id, statusChangeReason });
+        }
+        closeDialog();
+    };
+
+    const renderStatus = (report) => {
+        const { status } = report;
+        if (status === 'PENDING') {
+            return <Typography style={styles.pending}>{status}</Typography>;
+        }
+        if (status === 'CONFIRMED') {
+            return (
+                <Button
+                    size="small"
+                    style={styles.confirmed}
+                    onClick={() => openDialog(STATUS, report)}
+                >
+                    {status}
+                </Button>
+            );
+        }
+        if (status === 'REJECTED') {
+            return (
+                <Button
+                    size="small"
+                    style={styles.rejected}
+                    onClick={() => openDialog(STATUS, report)}
+                >
+                    {status}
+                </Button>
+            );
+        }
+        return <Typography style={styles.pending}>Invalid status</Typography>;
+    };
+
+    const statusContent = (
+        <>
+            <DialogTitle id="status-dialogue">Status change reason</DialogTitle>
+            <DialogContent>
+                <DialogContentText id="status-dialog-description">
+                    {selectedReport && selectedReport.status_change_reason}
+                </DialogContentText>
+            </DialogContent>
+            <DialogActions>
+                <Button onClick={closeDialog}>
+                  Close
+                </Button>
+            </DialogActions>
+        </>
+    );
+
+    const formContent = (
+        <>
+            <DialogTitle id="status-dialog-title">
+                {`${mode === REJECT ? 'Reject' : 'Confirm'} ${
+                    selectedReport && selectedReport.closure_state === CLOSED ?
+                        'closure' : 'reopening'
+                } of facility`}
+            </DialogTitle>
+            <DialogContent>
+                <TextField
+                    autoFocus
+                    margin="dense"
+                    id="status-change-reason"
+                    label="Status change reason"
+                    variant="outlined"
+                    multiline
+                    value={statusChangeReason}
+                    onChange={e => setStatusChangeReason(e.target.value)}
+                />
+            </DialogContent>
+            <DialogActions>
+                <Button onClick={closeDialog}>
+                  Cancel
+                </Button>
+                <Button onClick={updateReport} color="primary" variant="contained">
+                  Confirm
+                </Button>
+            </DialogActions>
+          </>
+    );
+
+    const statusDialog = (
+        <Dialog
```

Comment:
For consistency, we should consider copying the styling in `DashboardClaimDetailsControls.jsx` so that the approve/reject dialogs across different operations are consistent.

<img width="1349" alt="Screen Shot 2021-02-10 at 3 56 49 PM" src="https://user-images.githubusercontent.com/17363/107583842-c3ea5180-6bb8-11eb-9317-2de26fe19c9d.png">


<img width="1311" alt="Screen Shot 2021-02-10 at 3 58 45 PM" src="https://user-images.githubusercontent.com/17363/107583941-e41a1080-6bb8-11eb-989c-eb5034909a98.png">

Reply:
I brought over the style from `DashboardClaimDetailsControls.jsx` to make it more consistent. Once difference that remains is that the dialog here is narrower than the one in the Dashboard Claim Details, because the content is less wide. I added a `minWidth: 300px` to the text area, which is still a bit narrow, but I'm hesitant to make the minimum width larger in case it is too wide for narrower screens. 
I also made the `PENDING` status cell clickable to show the reason for reporting. I added an underline to the styling to be consistent with the other clickable items.
<img width="1240" alt="Screen Shot 2021-02-11 at 1 06 54 PM" src="https://user-images.githubusercontent.com/21046714/107680819-5697f880-6c6c-11eb-9a41-4ab4b9653b9f.png">